assets/js/pluginsidebar/components/sidebar/index.js

```diff
@@ -516,7 +516,13 @@ class Sidebar extends React.PureComponent {
       ? selectedSectionsRaw
       : [];
 
-    const parsedCoverArt = '' !== coverArt && 'null' !== coverArt
+    // Expects potentially an empty object,
+    // a string, or a null string from props.
+    // Contains object fallback if post type is not enabled,
+    // but still enables Gutenberg.
+    const parsedCoverArt = 0 !== Object.entries(coverArt).length
```

Comment:
Let's do a try/catch here instead to catch all instances in which it will fail and use a fallback.
```javascript
let parsedCoverArt;
try {
  parsedCoverArt = JSON.parse(coverArt);
} catch () {
  parsedCoverArt = {};
}
```

Reply:
Learned something there, this is fantastic.